.travis.yml

```diff
@@ -12,19 +12,19 @@ env:
 matrix:
   fast_finish: true
   include:
-    - php: 7.3
+    - php: 7.4
       env:
         - DEPS=lowest
-    - php: 7.3
+    - php: 7.4
       env:
         - DEPS=latest
         - CS_CHECK=true
         - BENCHMARKS=true
         - TEST_COVERAGE=true
-    - php: 7.4
+    - php: 8.0
```

Comment:
We should have this in allowed failures

Reply:
this is not working yet on travis, we have to use nightly if we want test on PHP 8.0

and of course we need much more changes in the travis to allow it to happen....